.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: Workflow for Codecov example-python
+on: [push, pull_request]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
```

Comment:
```suggestion
      - name: Set up Python 3.9
        uses: actions/setup-python@v2
        with:
          python-version: 3.9
      - name: Install dependencies
```

Reply:
Actually, I don't think this is necessary as we are using the `3.9` build anyways. Also, then we don't have to install deps